src/all_libs_classes_stylesheets.js

```diff
@@ -40,6 +40,7 @@
   appendScript('msg/js/en.js');
   appendScript('lib/blocks_compressed.js');
   appendScript('closure-library/closure/goog/base.js');  // Must be after Blockly
+  appendScript('src/controller/block_definition_extractor.js');
```

Comment:
This is our horrible way for getting the imports to work on both the app (mixed Node / web) and tests.